rollup.config.js

```diff
@@ -1,25 +1,44 @@
+import path from 'path';
+import { fileURLToPath } from 'url';
+import * as fs from 'node:fs';
 import { nodeResolve } from '@rollup/plugin-node-resolve';
 import commonjs from '@rollup/plugin-commonjs';
-import { terser } from 'rollup-plugin-terser';
+import terser from '@rollup/plugin-terser';
 
-export default {
-  input: './lib/svgo.js',
-  output: {
-    file: './dist/svgo.browser.js',
-    format: 'esm',
+const __dirname = path.dirname(fileURLToPath(import.meta.url));
+const pkgPath = path.join(__dirname, './package.json');
+const PKG = JSON.parse(fs.readFileSync(pkgPath, 'utf-8'));
+
+export default [
+  {
+    input: './lib/svgo.js',
+    output: {
+      file: './dist/svgo-node.cjs',
+      format: 'cjs',
+    },
+    external: Object.keys(PKG.dependencies),
+    onwarn(warning) {
+      throw Error(warning.toString());
+    },
+    plugins: [terser()],
```

Comment:
Shouldn't this use the same options as the esm config?

Reply:
Thanks for calling that out, you're right, though I do have some thoughts here.

* We should keep `mangle: false`. ✅
* I strongly disagree with how we have `comment: false` presently, it should be `'some'` (default) to preserve comments regarding license, copyright, and attribution.
* Reviewing the docs, compress can hinder readability, but rather than turn it off entirely, we could just disable the rules that may impact have a negative impact and keep the smaller changes.

Later this week, I'll review the options for the terser plugin and see what we should do. I may revise `compress` and `format` to fulfil the motivation of the current settings (minify while retaining debug potential).

Related: https://github.com/svg/svgo/pull/1417

By any chance, do you personally utilize or benefit from the reduced minification of these bundles?

So far, thinking a config like this:

```js
/**
 * @typedef {import('@rollup/plugin-terser').Options} Options
 */

/** @type {Options} */
const terserOptions = {
  compress: {
    defaults: false,
    arrows: true,
    computed_props: true,
    conditionals: true,
    dead_code: true,
    drop_debugger: true,
    evaluate: true,
  },
  mangle: false,
  format: { 
    comments: 'some',
    keep_numbers: true,
    semicolons: false,
    shebang: false
  },
};
```

| File | `main` | Current Browser Options | Proposed Options |
|---|---|---|---|
| `svgo.browser.js` | 754.2 kB | 754.2 kB | 749.8 kB | 
| `svgo-node.cjs` | 123.7 kB | 184.5 kB | 182.8 kB |

Reply:
My point was that one of the two bundles didn't use the same options.

The options is another discussion :)

Thanks for looking into it!